### PR TITLE
feature/rabbitmq-kubernetes-operator

### PR DESCRIPTION
**Pull request by apolloakora**
Replacing the RabbitMQ cluster setup to use Kubernetes Operator instead of the deprecated helm chart.

### CODE REVIEW

**Comment by apolloakora** [comment]
[Replace RabbitMQ Helm Chart with Kubernetes Operator](https://trello.com/c/KSy7A1CK/1526-replace-rabbitmq-helm-chart-with-kubernetes-operator)

File: run_in_kubernetes.sh [hunk 1]
```diff
@@ -11,9 +11,9 @@ kubectl run sampleloader -it --rm \
    --image $IMAGE\
    --env=RABBITMQ_USER=$(kubectl get secret rabbitmq -o=jsonpath="{.data.rabbitmq-username}" | base64 --decode) \
    --env=RABBITMQ_PASSWORD=$(kubectl get secret rabbitmq -o=jsonpath="{.data.rabbitmq-password}" | base64 --decode) \
-   --env=RABBITMQ_SERVICE_HOST=rabbitmq \
+   --env=RABBITMQ_SERVICE_HOST=rm-rabbitmq-client \
```
Comment (NickGrantX) [inline run_in_kubernetes.sh]: this is an extremely odd name for the hostname of the Rabbit service. _We_ are the client and Rabbit is the host

Comment (NickGrantX) [reply run_in_kubernetes.sh]: I think this is plain wrong. It's `rm-rabbitmq` everywhere else

**Review by NickGrantX — CHANGES_REQUESTED** [review]
See comment

File: run_in_kubernetes.sh (hunk 1 above)
Comment (apolloakora) [reply run_in_kubernetes.sh]: Good point - that is probably why the operator people decided to change it and remove the client bit.

**Review by AdamHawtin — APPROVED** [review]
I don't think this script should be used for anything at all any more but this looks like the correct change to keep it working so LGTM until we get round to deleting it 👍

**Review by NickGrantX — APPROVED** [review]
Looks good

**Pull request merged**